test_data/rules/prefer_final_parameters.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `dart test -N prefer_final_parameters`
+
+void badMethod(String label) { // LINT
```

Comment:
nit: this is a top-level function, not a method

Reply:
Good catch, thanks!